plugins/module_utils/network/vyos/rm_templates/bgp_address_family.py

```diff
@@ -234,6 +255,32 @@ def _tmplt_bgp_af_neighbor(config_data):
     return command
 
 
+def _tmplt_bgp_af_network_generic(config_data):
```

Comment:
Do you have tests for cases where you might have values for both backdoor and route-map?  If the rm templating system calls this method for both, how are you going to tell which one to emit?

Reply:
I will double check, however, I believe we added to the command dict based on the parser output, and the parsers (and Ansible dicts) are different. Utimately, we can add some more tests, even though, I was not changing this particular funcionality but rather levveraging what was there